docs/dr.md

```diff
@@ -0,0 +1,152 @@
+# DR Commands
+
+The dr command supports the following sub-commands:
+
+* [init](#init)
+* [test](#test)
+
+## init
+
+The init command crates a configuration file required for all other dr commands.
+
+```bash
+$ odf dr init
+✅ Created config file "config.yaml" - please modify for your clusters
+```
+
+> [!IMPORTANT]
+> Before using the config file, you need to edit it and configure your
+> clusters and storage.
+
+### Sample configuration file
+
+```yaml
+## odf dr configuration file
+
+## Clusters configuration.
+# - Modify clusters "kubeconfig" and "name" to match your hub and managed
+#   clusters names and path to the kubeconfig file.
+clusters:
+  hub:
+    name: hub
+    kubeconfig: /Users/nir/.config/drenv/rdr/kubeconfigs/hub
+  c1:
+    name: dr1
+    kubeconfig: /Users/nir/.config/drenv/rdr/kubeconfigs/dr1
+  c2:
+    name: dr2
+    kubeconfig: /Users/nir/.config/drenv/rdr/kubeconfigs/dr2
```

Comment:
This is modified config for ramen testing env, not the default config. Need to update to show the defaults.